package.json

```diff
@@ -108,8 +108,8 @@
     "aws-sdk": "^2.854.0",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "dcl-catalyst-client": "^3.2.0",
-    "dcl-catalyst-commons": "^2.1.0",
+    "dcl-catalyst-client": "^4.0.3-20210322145441.commit-8c2b9c7",
```

Comment:
This is on purpose. Once [this change](https://github.com/decentraland/catalyst-client/pull/75) is tested and merged, we will release a new version of the library and then update this dependency